groups/instance/asg.tf

```diff
@@ -68,9 +68,9 @@ resource "aws_autoscaling_attachment" "artifactory_asg_attachment" {
 }
 
 resource "aws_autoscaling_schedule" "scale_down" {
-  desired_capacity       = var.asg_desired_capacity
-  min_size               = var.asg_min_size
-  max_size               = var.asg_max_size
+  desired_capacity       = var.asg_scale_down_desired_capacity
+  min_size               = var.asg_scale_down_min_size
+  max_size               = var.asg_scale_down_max_size
```

Comment:
Just a suggestion, but you could add a timezone to the scale_down and up otherwise it will shift by an hour when daylight saving comes/goes - e.g. https://github.com/companieshouse/terraform-modules/blob/main/aws/ecs/ecs-cluster/ec2.tf#L116

Reply:
modified to use var.asg_time_zone